package.json

```diff
@@ -25,5 +25,8 @@
     "concurrently": "^6.3.0",
     "husky": "^7.0.4"
   },
-  "dependencies": {}
-}
\ No newline at end of file
+  "dependencies": {
```

Comment:
I think it'd be better to put these in `frontend/package.json`

Reply:
Yep; in fact, they were already there, so I just deleted it from package.json